testnet/launcher/docker.go

```diff
@@ -62,8 +62,6 @@ func (t *Testnet) Start() error {
 		node.WithDebugNamespaceEnabled(true),
 		node.WithLogLevel(4),
 		node.WithEdgelessDBImage("ghcr.io/edgelesssys/edgelessdb-sgx-4gb:v0.3.2"), // default edgeless db value
-		node.WithBatchInterval("1s"),
```

Comment:
These values are defaulted in the node config so this was redundant, (and the linter was getting upset at how much duplication there was in here).